snapcraft/plugins/maven.py

```diff
@@ -103,18 +114,22 @@ def build(self):
 
         self.run(mvn_cmd + self.options.maven_options)
 
-        jarfiles = glob.glob(os.path.join(self.builddir, 'target', '*.jar'))
-        warfiles = glob.glob(os.path.join(self.builddir, 'target', '*.war'))
-        if not (jarfiles or warfiles):
-            raise RuntimeError('could not find any built jar files for part')
-        if jarfiles:
-            jardir = os.path.join(self.installdir, 'jar')
-            os.makedirs(jardir, exist_ok=True)
-            self.run(['cp', '-a'] + jarfiles + [jardir])
-        if warfiles:
-            wardir = os.path.join(self.installdir, 'war')
-            os.makedirs(wardir, exist_ok=True)
-            self.run(['cp', '-a'] + warfiles + [wardir])
+        for f in self.options.maven_targets:
+            src = os.path.join(self.builddir, f, 'target')
+
+            jarfiles = glob.glob(os.path.join(src, '*.jar'))
+            warfiles = glob.glob(os.path.join(src, '*.war'))
+            arfiles = glob.glob(os.path.join(src, '*.[jw]ar'))
+
+            if not (arfiles):
+                raise RuntimeError("could not find any"
+                                   "built jar files for part")
+            basedir = 'jar' if jarfiles and len(f) == 0 else (
+                      'war' if warfiles and len(f) == 0 else f)
```

Comment:
Where can I find any docs on this, it was pretty rough to understand what was there to begin with.